cni/pkg/ebpf/server/redirectionServer_linux.go

```diff
@@ -686,6 +686,17 @@ func (r *RedirectServer) DumpAppIPs() sets.String {
 	return m
 }
 
+func (r *RedirectServer) IsPodIPEnrolled(ip string) bool {
+	var valueOut mapInfo
+	ipAddr, err := netip.ParseAddr(ip)
+	if err != nil {
+		return false
+	}
+	err = r.obj.AppInfo.Lookup(ipAddr.As4(), &valueOut)
```

Comment:
nit: return r.obj.AppInfo.Lookup == nil

Does it only return err when the IP not found?

Reply:
Updated